src/components/inputs/date-input/date-input.mod.css

```diff
@@ -1,3 +1,81 @@
-.container {
-  width: 100%;
+ .headingControls {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-size: 13px;
+  margin: 8px 16px;
+  padding: 0 2%;
+}
+
+.month {
+  flex: 1 1 auto;
+  text-align: center;
+  color: var(--token-font-color-default);
+  font-size: var(--token-font-size-default);
+}
+
+.daysHeaderContainer {
+  padding: var(--spacing-16) var(--spacing-8);
+  border-top: 1px solid var(--color-gray-90);
+}
+
+.daysHeader {
+  padding-left: 2%;
+}
+
+.controlContainer {
+  display: flex;
+}
+
+.controlCalendar {
+  border: 1px solid var(--token-border-color-input-pristine);
+  border-radius: var(--token-border-radius-input);
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+  border-left: 0;
+  padding: 0 8px;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.controlCalendarDisabled {
+  composes: controlCalendar;
+  border-color: var(--token-border-color-input-disabled);
+  background-color: var(--token-background-color-input-disabled);
+}
+
+.controlCalendarError {
+  composes: controlCalendar;
+  border-color: var(--token-border-color-input-error);
+}
+
+.controlCalendarWarning {
+  composes: controlCalendar;
+  border-color: var(--token-border-color-input-warning);
+}
+
+.controlCalendarFocused {
+  composes: controlCalendar;
+  border-color: var(--token-border-color-input-focus);
+}
+
+.daysHeaderItem {
+  color: var(--color-gray-60);
+  font-weight: bold;
+  cursor: default;
+  font-size: var(--token-font-size-small);
+  display: inline-block;
+  width: 12%;
```

Comment:
How do these values come about? Could we rather `calc` them?

Reply:
This is for styling the inputs. The calendar dropdown has a flexible width which depends on the `horizontalConstraint`. The days always fill the whole calendar.

![image](https://user-images.githubusercontent.com/1765075/48352566-44920e80-e68d-11e8-8aac-e6b3badd0157.png)

![image](https://user-images.githubusercontent.com/1765075/48352571-4b208600-e68d-11e8-8816-25d0ed2bf6c1.png)

The percentages are used to format the days in the calendar. The specific values ensure that the days are always centered and fill the whole screen.

The reason for these values is that: there are 7 days in a week. 7*12% = 84%. There are 6 spaces between the 7 days. Each day has 1% margin on the left and on the right: 6 * 2 * 1% = 12%. This totals 96%. Then we add 2% spacing to the left and 2% spacing to the right of the calendar and we end up with a calendar having 100% width.

There is not really a way to `calc` them as so many rely on each other and it only adds up to 100 for certain combinations.